src/Sulu/Bundle/AdminBundle/Resources/js/views/Form/toolbarActions/TypeToolbarAction.js

```diff
@@ -1,15 +1,30 @@
 // @flow
+import jexl from 'jexl';
 import type {ToolbarItemConfig} from '../../../containers/Toolbar/types';
 import AbstractFormToolbarAction from './AbstractFormToolbarAction';
 
 export default class TypeToolbarAction extends AbstractFormToolbarAction {
-    getToolbarItemConfig(): ToolbarItemConfig {
+    getToolbarItemConfig(): ?ToolbarItemConfig {
         const formTypes = this.resourceFormStore.types;
 
         if (!this.resourceFormStore.typesLoading && Object.keys(formTypes).length === 0) {
             throw new Error('The ToolbarAction for types only works with entities actually supporting types!');
         }
 
+        const {
+            visible_condition: visibleCondition,
```

Comment:
Actually I am not sure if this one makes sense here at all... If we do not display the type dropdown all together, then people without this `ToolbarAction` won't see what type is selected.